keps/sig-auth/5018-dra-adminaccess/README.md

```diff
@@ -179,9 +179,11 @@ objects as privileged. This feature includes:
    ```yaml
    metadata:
      labels:
-       resource.k8s.io/admin-access: "true"
+       resource.kubernetes.io/admin-access: "true"
    ```
 
+   Note: This label has been updated from `resource.k8s.io/admin-access` while the feature was in alpha in v1.33.
```

Comment:
This is not quite accurate, b/c 1.33 is still using `resource.k8s.io/admin-access: "true"`, so maybe
```suggestion
   Note: This label has been updated from `resource.k8s.io/admin-access` before the beta promotion.
```
or
```suggestion
   Note: This label has been updated from `resource.k8s.io/admin-access` while the feature was in alpha.
```
Ideally I'd say open a PR doing so asap.